.github/workflows/plugins.yml

```diff
@@ -943,7 +959,7 @@ jobs:
       - uses: ./.github/actions/testagent/start
       - uses: ./.github/actions/node/setup
       - run: yarn install
-      - uses: ./.github/actions/node/oldest
+      - uses: ./.github/actions/node/18
```

Comment:
Doesn't that make it so that we don't know if we support Node 14 for Next? (and other similar ones)

Reply:
Correct. In these cases, the tests do not pass on older Node.js versions, so we can't verify (or perhaps even _claim_) support. Arguably, we never could.